tls/s2n_config.c

```diff
@@ -68,14 +68,12 @@ static struct s2n_config s2n_default_tls13_config = {0};
 static int s2n_config_setup_default(struct s2n_config *config)
 {
     GUARD(s2n_config_set_cipher_preferences(config, "default"));
-    GUARD(s2n_config_set_ecc_preferences(config, "default"));
     return S2N_SUCCESS;
 }
 
 static int s2n_config_setup_tls13(struct s2n_config *config)
 {
-    GUARD(s2n_config_set_cipher_preferences(config, "default_tls13"));
-    GUARD(s2n_config_set_ecc_preferences(config, "default_tls13"));      
+    GUARD(s2n_config_set_cipher_preferences(config, "default_tls13"));    
```

Comment:
hmm i wonder if this is still needed anymore

Reply:
It would be required as this is the place that the configurations for default tls13 is setup when s2n_init() is called.